rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/Fixture/sensio_security.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Fixture;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+
+class SensioSecurity
+{
+    /**
+     * @Security("is_granted('ROLE_USER')")
+     */
+    public function action()
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Fixture;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+
+class SensioSecurity
+{
+    #[\Sensio\Bundle\FrameworkExtraBundle\Configuration\Security('is_granted(\'ROLE_USER\')')]
```

Comment:
Would be cool if Rector would change it to this instead. Any suggestions on how to do it quickly?

```suggestion
    #[\Sensio\Bundle\FrameworkExtraBundle\Configuration\Security("is_granted('ROLE_USER')")]
```

Reply:
From top of my head:

```php
$string->setAttbibute('kind', String_::KIND_xxx);
```